k8s-tests/chainsaw/helm/install-helm-chart.sh

```
@@ -28,5 +28,28 @@ else
     HELM=$(which helm)
 fi
 
+LOCAL_OPERATOR_IMG_ARGS=()
+if [ -n "${LOCAL_OPERATOR_IMG:-}" ]; then
+    LOCAL_OPERATOR_IMG_WITHOUT_DIGEST=${LOCAL_OPERATOR_IMG%%@*}
+    LOCAL_OPERATOR_IMG_DIGEST=""
+    if [[ "${LOCAL_OPERATOR_IMG}" == *@* ]]; then
+        LOCAL_OPERATOR_IMG_DIGEST=${LOCAL_OPERATOR_IMG#*@}
```

Shellcheck (0.11.0)

[warning] LOCAL_OPERATOR_IMG_DIGEST appears unused. Verify use (or export if used externally).

(SC2034)